lib/ifupdown.sh.in

```diff
@@ -1,11 +1,15 @@
 #!/bin/sh
 
+# This script should be symlinked into /etc/network/ip-pre-up.d/ and
```

Comment:
i also don't have any preferred style for shell scripts ; 
but we will use this as the project's preferred style ; 
i might have to write a style doc ;

will also check out the ` http://arachnoid.com/python/beautify_bash_program.html` tool

thanks